confluence-copy-link-buttons.user.js

```diff
@@ -114,7 +114,7 @@
 					return;
 				}
 			}
-			const url = document.location.href;
+			const url = document.querySelector('link[rel="shortlink"]').href;
```

Comment:
The argumentation in the commit message might be true, but in my ten years of using Confluence, I find that having the original page name in the URL is more useful. The sharelinks aren't as stable as one might think – if two Confluence instances are merged in some way, the sharelinks usually become dead.

Reply:
...and finding the new location post-merge is relatively easy with page-name-link, while almost (?) impossible with the shortlink.